src/main/kotlin/platform/mixin/reference/MixinSelectors.kt

```diff
@@ -387,6 +389,25 @@ private class MixinRegexSelector(
 
 // Dynamic selectors
 
+/**
+ * A dynamic selector can change which class to query for members if desired.
+ */
+interface DynamicMixinSelector : MixinSelector {
+    fun redirectOwner(owner: ClassNode): ClassNode {
```

Comment:
I think it's fine for this method to be moved directly into the `MixinSelector` interface. That would remove the need for the static method below. Also maybe the method could be called `getCustomOwner` or something?